WebView/WebkitWebView/spotless/copyright.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2024 The Android Open Source Project
```

Comment:
Can you check here on how to add this? Year should be variable and take current year only. https://github.com/android/identity-samples/tree/main/CredentialProvider/MyVault/spotless